src/index.html

```diff
@@ -7,11 +7,158 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Personal information</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form tabindex="1">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input type="text"
+        name="usersurname"
+        autocomplete="off"
+        ><br>
+
+        <label for="name">Name:</label>
+        <input type="text"
+        name="username"
+        autocomplete="off"
+        ><br>
+
+        <label for="old"> How old are You?</label>
+        <input type="number"
+        name="userage"
+        min="1"
+        max="100"
+        value="12"
+        autocomplete="off"
+        ><br>
+
+        <label for="old"> Full date of birth:</label>
+        <input type="date"
+        name="userbirth"
+        ><br>
+
+        <label for="old"> I accept the term of the agreement</label>
+        <input type="checkbox"
+        name="iacc"
+        ><br>
+      </fieldset>
+    </form>
+
+
+    <form tabindex="2">
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="surname">E-mail:</label>
+        <input type="email"
+         name="email"
+         autocomplete="off"
+         placeholder="email@example.com"
+        ><br>
+
+        <label for="password">Password:</label>
+        <input type="password"
+        name="password"
+        autocomplete="off"
+        >
+      </fieldset>
+    </form>
+
+
+    <form tabindex="3">
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label>
+          <label for="cats">Do you love cats?</label>
+          <input type="radio"
+           name="cat"
+           value="yes"
+          > Yes
+        </label>
```

Comment:
no needs to add nested `label` like this
Use labels for inputs and move the text `Do you love cats?` outside the label
```suggestion
        <label>
          <label for="cats">Do you love cats?</label>
          <input type="radio"
           name="cat"
           value="yes"
          > Yes
        </label>
```